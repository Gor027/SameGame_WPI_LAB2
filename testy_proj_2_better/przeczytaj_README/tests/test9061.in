T Tu  T  u uuTuTu uTT  T uu uT u
T u  TTT    T T uT u  TTuu Tu  u T uuT   u Tu     Tu    uuT u T
TTuuuTuuuTuT T   TTuTTuuuuT u
uuu u  TTuT u uuuTuu T   TT uTT
u  Tu TuTTT
  TuTu u TTuTuT uuu  TTT T  uuuTuuTTTT  u TuuTT u  T
 uu  TTu T  u Tuuu uuT  uuT TT
Tu  TT uu TuT u uTu   uT Tu uu
u TT u uT     Tu T  uuu u TTT uuu uuTu uTu  T T
T   uuuuTu TuTuuT TuT uTTT TTTTTTT uTTTT  T
Tuuuuuuuu uuTTuuuu uuu  u  Tu T u TTTuTuu   TTTu u u   TuuTTuu T  T TuTuT T
TTuTuuTu Tuu  uuTuu T  T  uTuu uuu T  uuT T uuT u u TT   T Tu   
u   TuTT 
   uuT TTuuTu TuT T  Tu TuTTu  u T uuuTuT
