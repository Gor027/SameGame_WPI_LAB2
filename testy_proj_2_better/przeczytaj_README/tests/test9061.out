       T
       u                    u
     T T      u             u
T   uT u      T u           u
T T uuTu u  u u uT       u  u
T uuuTTuuTuuT TTuTu      uu u
u uuTTuuTuTTu TuuTTTTTTTuuT TT u   u
uTuTTuTuTTTuT uuuuuuTTuTuTTTuT u   TuT
TuTuuuTuuTTuuTTuuuuuuTTuuTTTTuTTuT uTT uu   u
uuuTuuuuTTTTTTuuTTuuuTuTuuuTTTuuuu uTuTTT   u T
TuTuuuuuTuuuuuuuuTTuTuuTTTTTuuuTTTTTTTTuu T TTT    u    u  u
TuuTTuTTuTuuTTuuTuuuuuuTuTuTuTTuuuuTTuuuTuTTTTT u TT   uuTTu
TTuuuTTTTuuTuTTuTuTuTTuTTuTTuuTuuTTuuuTuTuTuuuTuuuuuTTTuTTTuuTTTTuTuTT
